CHANGELOG.md

```diff
@@ -262,3 +262,6 @@
 * C4 for Quiet architecture. Context and Container diagrams.
 
 * Invite tab as default in settings
+
+[2.0.0-alpha.12]
+* Refactored package.json to have consistent license "GPL-3.0-or-later" 
```

Comment:
Would you mind moving this up to the top under unreleased?

Reply:
@leblowl could you pls check now? does this work

Reply:
@bhavyagor12 one more small suggestion